sonar-project.properties

```diff
@@ -15,7 +15,7 @@ sonar.sources = ansible_ai_connect/,ansible_ai_connect_admin_portal/,ansible_ai_
 sonar.tests = ansible_ai_connect/,ansible_ai_connect_admin_portal/,ansible_ai_connect_chatbot/
 
 # Include test subdirectories in test scope
-sonar.test.inclusions = ansible_ai_connect/**/test_*.py,ansible_ai_connect/users/migrations/*,ansible_ai_connect/organizations/migrations/*,ansible_ai_connect_admin_portal/**/__tests__/,ansible_ai_connect_chatbot/src/**
+sonar.test.inclusions = ansible_ai_connect/**/test_*.py,ansible_ai_connect/users/migrations/*,ansible_ai_connect/organizations/migrations/*,ansible_ai_connect_admin_portal/**/__tests__/
```

Comment:
Erm, doesn't this now remove Chat Bot from SonarClouds *inclusions*?

Reply:
I initially thought this was for specifying the location of test files, but now I am suspecting it is causing this error...
![image](https://github.com/user-attachments/assets/1a7af1dc-0348-443e-ae1d-29f6adbb86e8)

Reply:
Now the `App.test.tsx` shows up in the coverage list... I need to get rid of that.
![image](https://github.com/user-attachments/assets/0705ae82-007a-4c87-a969-e6c0b4efe280)

Reply:
![image](https://github.com/user-attachments/assets/6ddff976-75e2-4557-b506-56ef135bb08c)

`App.test.tsx` was removed.  We still have no coverage on `index.tsx` and `reportWebVitals.ts`, but I think >90% would be good enough.